src/plugins/vis_type_table_new/public/components/table_vis_component.tsx

```diff
@@ -51,8 +52,12 @@ export const TableVisComponent = ({
       const colIndex = columns.findIndex((col) => col.id === columnId);
       const column = columns[colIndex];
       // use formatter to format raw content
-      // this can format date and percentage data
-      const formattedContent = column.formatter.convert(rawContent, 'text');
+      // this can format url, date and percentage data
+      const htmlContent = column.formatter.convert(rawContent, 'html');
+      const formattedContent = (
+        // eslint-disable-next-line
```

Comment:
Instead of completely disabling eslint for the line, we should use the specific rule and add an explanatory comment (see comment below).

```
<div dangerouslySetInnerHTML={{ __html: dompurify.sanitize(htmlContent) }} /> // eslint-disable-line react/no-danger
```

Reply:
will add this. thx